src/main/java/mate/academy/bookstore/anotation/FieldMatchValidator.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.bookstore.anotation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.util.Objects;
+import mate.academy.bookstore.dto.user.UserRegistrationRequestDto;
+
+public class FieldMatchValidator implements
+        ConstraintValidator<FieldMatch, UserRegistrationRequestDto> {
+    @Override
+    public boolean isValid(UserRegistrationRequestDto requestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return Objects.equals(requestDto.getPassword(), requestDto.getRepeatPassword());
+    }
```

Comment:
not resolved, parameterize this implementation of the `ConstraintValidator` with `Object` instead of `UserRegistrationRequestDto` to make it universal and so that it could work with any class, by using reflection to get the declared fields of the class, and then values of  those fields to compare them